<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>1981</width>
      <height>1433</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>네이버 지식인 크롤링</string>
     </attribute>
     <widget class="QLineEdit" name="ed_search">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>46</y>
        <width>509</width>
        <height>73</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>754</x>
        <y>44</y>
        <width>201</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>48</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>네이버검색 : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>158</y>
        <width>261</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>저장파일명</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_result">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>218</y>
        <width>717</width>
        <height>83</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>워드클라우드</string>
     </attribute>
     <widget class="QLineEdit" name="ed_file">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>44</y>
        <width>415</width>
        <height>67</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>46</y>
        <width>155</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>파일선택</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_stopword">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>124</y>
        <width>417</width>
        <height>73</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>126</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>금지어</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_process">
      <property name="geometry">
       <rect>
        <x>612</x>
        <y>42</y>
        <width>343</width>
        <height>153</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>분석</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_word">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>238</y>
        <width>899</width>
        <height>583</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string> 워드클라우드 결과보기 화면</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>1818</x>
      <y>1452</y>
      <width>175</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>프로그램 종료</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
